packages/nc-gui/components/ProjectTreeView.vue

```diff
@@ -963,7 +985,90 @@ export default {
     openLink(link) {
       window.open(link, '_blank');
     },
-
+    async checkAndDeleteTable(table) {
```

Comment:
I have duplicated this function from `components/project/spreadsheet/rowsXcDataTable.vue`.